xbmc/video/tags/VideoInfoTagLoaderFactory.cpp

```diff
@@ -23,7 +24,7 @@ IVideoInfoTagLoader* CVideoInfoTagLoaderFactory::CreateLoader(const CFileItem& i
                                                               bool forceRefresh)
 {
   // don't try to read tags for streams
-  if (item.IsInternetStream())
+  if (item.IsInternetStream() && !URIUtils::IsOnLAN(item.GetPath()))
```

Comment:
better use GetDynPath()

Reply:
Have made change. Not sure why jenkins is failing. Builds and tests ok locally.